### DISCUSSION TITLE
Detailed Setup menus are different on HD200/300

### DISCUSSION THREAD

**Discussion by metapath-zz** [OpenSageTV/Community#137] [General]
On my SageTV server (Windows 10), the Detailed Setup menus are different when I run a client on the server than when using either of my HD200/300 boxes. Specifically, I needed to change the audio delay in the Video/Audio menu, but this menu is only available from a client running on the server. Anybody know why? Can I add this menu so it appears from a connected HD200/300?

**Comment by xjim1** [comment]
I assume when using your HD device, you are running a client on the connected system?
I honestly don't know much of anything about it, but since activity here is pretty sparse, I'll help out as I can.

**Reply by xjim1** [reply]
Ok scratch that.  I just read up on what an HD device really does. :\
